tests/ci/common_posix_setup.sh

```diff
@@ -1,15 +1,19 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR ISC
 
+SRC_ROOT="$(pwd)"
 if [ -v CODEBUILD_SRC_DIR ]; then
   SRC_ROOT="$CODEBUILD_SRC_DIR"
-else
-  SRC_ROOT=$(pwd)
+elif [ "$(basename "${SRC_ROOT}")" != 'aws-lc' ]; then
+  SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+  SRC_ROOT="${SCRIPT_DIR}/../.."
 fi
+SRC_ROOT="$(readlink -f "${SRC_ROOT}")"
 echo "$SRC_ROOT"
 
-cd ../
-SYS_ROOT=$(pwd)
+SYS_ROOT="$(readlink -f "${SRC_ROOT}/..")"
+echo "$SYS_ROOT"
```

Comment:
nit: here and L12, should we echo the variable name as well? otherwise it's just some paths outputted to stdout...


```
echo "SYS_ROOT $SYS_ROOT"
```